### ISSUE TITLE
White image is getting grey edges when rendered

### ISSUE THREAD

**Issue by ivan-i1** [mrdoob/three.js#14668]
##### White image is getting grey edges when rendered

I first noticed this grey-edge thing when using a white materials on custom meshes. I was blaming the geometries but then I tried to render a white image in the most straight-forward way I know without any special meshes and I got the grey edges. I also tried rendering a full opacity image (no antialias or anything else) and that one also got the grey edges. Is this  known issue?

Below the white PNG image I used for testing.
![snowflake](https://user-images.githubusercontent.com/3728845/43851107-624deb6c-9aff-11e8-8f92-6b945e38a408.png)

Here a screenshot of the codepen with the snowflake being rendered on a white background.
(Please note how the grey edges make the snowflake visible)
![screen shot 2018-08-08 at 11 37 34 am](https://user-images.githubusercontent.com/3728845/43851152-7e9f1d0e-9aff-11e8-81ab-e7a297fde49c.png)

* [codepen](https://codepen.io/ivan-i1/pen/pZxwZX) (release branch 87)

##### Three.js version
- [x] 87

##### Browser
- [x] Chrome
- [x] Firefox
- [x] Safari

##### OS
- [x] macOS

**Comment by WestLangley** [comment]
Please do not cross-post stackoverflow help question here.

You can try the [forum](https://discourse.threejs.org) if you want.

**Comment by ivan-i1** [comment]
Ok, let me rephrase and sorry in advance if I seem stubborn.

I have a perfectly white image.
Using the last ThreeJS release (95) and the 87 version for the example only (no additional libraries or anything).
I render this image using only ThreeJS in the path:
Image->Texture->Material->Render
The render has white background.

So there is no reason for other than white pixels to appear.
But they do.

I'm claiming here that this is a bug, are you already aware of it?

**Comment by mrdoob** [comment]
Can you please post this in the [forum](https://discourse.threejs.org/) instead?

**Comment by WestLangley** [comment]
Mipmapping is most likely the cause of your issue.

You can turn off mipmapping:
```javascript
texture.generateMipmaps = false;
texture.magFilter = THREE.LinearFilter;
texture.minFilter = THREE.LinearFilter;
```

Alternatively,  you can premultiply alpha on upload to the GPU, but that requires that you specify a custom blending function. Assuming you still want normal alpha blending, you would do this:
```javascript
texture.premultiplyAlpha = true;

material = new THREE.MeshBasicMaterial( {
    map: texture,

    blending: THREE.CustomBlending,
    blendEquation: THREE.AddEquation,
    blendSrc: THREE.OneFactor,
    blendDst: THREE.OneMinusSrcAlphaFactor,
    blendSrcAlpha: THREE.OneFactor,
    blendDstAlpha: THREE.OneMinusSrcAlphaFactor,

    transparent: true
} );
```

See https://developer.nvidia.com/content/alpha-blending-pre-or-not-pre.
